userdocs/profiles.dev/src/components/Schema.js

```diff
@@ -0,0 +1,254 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Modifications:
+  Copyright 2021 Weaveworks
+*/
+
+import React, {useState, useEffect, useRef} from 'react';
+import styles from '../css/schema.module.css';
+import { html, render } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+
+// TODO make the styling work
+
+function Schema(input) {
```

Comment:
thanks for making this working with various json files, love it. I'd prefer if you move this whole function block to the end of the file , just above line 254 or you can change this to be code suggested and remove line 254 💡 

```suggestion
export default function Schema(input) {
```

Reply:
> move this whole function block to the end of the file

is that javascript practice then? to have exported things at the end?

Reply:
well afaik you can export anywhere if you use `export default function Schema(input) {`  but since you are exporting schema function at line 254 then it makes sense to keep line 254 and the  `function Schema(input) {` next to each other